test-packages/test-services-odata-v2/generate-odata-services.ts

```diff
@@ -7,7 +7,7 @@ const logger = createLogger('generate-odata-services');
 const generatorConfigOData = {
   forceOverwrite: true,
   overwrite: true,
-  generateJs: false,
+  transpile: false,
```

Comment:
[pp] You could shorten the list of properties by using the default.

Reply:
I realized `generateJs` is set false here and then later to true in `generateOdata()`. Set `transpile` to true here (and only once)